src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -327,6 +327,58 @@
         component.set('v.data', rows);
     },
 
+    /**
+     * @description: Dry run all data import records for batch
+     */
+    batchDryRun: function (component) {
+        let afterDryRun = function (result) {
+            helper.afterDryRun(component, JSON.parse(result, false));
```

Comment:
@ptewson-sfdo - is the "false" inside the JSON.parse necessary here?

Reply:
Good question @psadie - I don't THINK that it is needed and having looked it up, it seems like that second argument is for a reviver function so not super clear if the false is even intentional or not? But it's weird that it would be there if it didn't have a reason to be - @bethbrains or @kenslewis do you have any insight there? 🤔